libs/design/scss/layout/_variables.scss

```diff
@@ -1,8 +1,13 @@
-//
-// @docs
-//
-// Sensible breakpoints to be used for layouts and interfaces.
-// These breakpoints are based on common devices viewport widths.
+/// Defines sensible breakpoints for layouts and interfaces.
+/// These breakpoints correspond to common device viewport widths,
+/// making it easier to apply responsive styles consistently across devices.
+///
+/// @group Layout
+///
+/// @example scss
```

Comment:
This example doesn't align with the token.

Reply:
How should I document this token then? Because it's only used in the context of the breakpoint mixin.